source/user-manual/api/equivalence.rst

```diff
@@ -111,7 +111,9 @@ The following tables contain the equivalencies between old API 3.x and API 4.1 e
-| Delete agents                                    | ``DELETE /agents``                                            | :api-ref:`DELETE /agents <operation/api.controllers.agent_controller.delete_agents>`                                                | Use the ``agents_list`` parameter instead of ``ids`` to indicate which agents must be deleted.                                                                                                 |
+|                                                  |                                                               |                                                                                                                                     | Use the ``agents_list`` parameter instead of ``ids`` to indicate which agents must be deleted.                                                                                                 |
+| Delete agents                                    | ``DELETE /agents``                                            | :api-ref:`DELETE /agents <operation/api.controllers.agent_controller.delete_agents>`                                                |                                                                                                                                                                                                |
+|                                                  |                                                               |                                                                                                                                     | Now the ``status`` parameter is mandatory. Its value must be one of the following: ``all``, ``active``, ``pending``, ``disconnected``, ``never_connected``.                                    |
 |                                                  |                                                               |                                                                                                                                     |                                                                                                                                                                                                |
 |                                                  |                                                               |                                                                                                                                     | If no ``agents_list`` is provided no agents will be removed. In order to remove all agents, it must be specified with the ``all`` keyword.                                                     |
```

Comment:
For **agents_list** and **status** keyword

Reply:
Changed